src/app/pages/dashboard/components/dashboard/dashboard.component.ts

```diff
@@ -145,4 +150,24 @@ export class DashboardComponent implements OnInit {
         this.renderedGroups.set(groups);
       });
   }
+
+  private updateIsEditingState(value: boolean): void {
```

Comment:
It may help avoid unnecessary wrapper functions https://angular.dev/guide/signals#effects